SK.Application/Articles/Queries/ListArticle/ListArticleQueryHandler.cs

```diff
@@ -15,32 +16,18 @@ namespace SK.Application.Articles.Queries.ListArticle
 { 
     public class ListArticleQueryHandler : IRequestHandler<ListArticleQuery, PagedResponse<List<ArticleDto>>>
     {
-        private readonly IApplicationDbContext _context;
-        private readonly IMapper _mapper;
-        private readonly IUriService _uriService;
+        private readonly IPaginationService<Article, ArticleDto> paginationService;
```

Comment:
Prefere to use underline before private field to avoid use "this." in constructor.